### PR TITLE
Use constant for keyword name in KeywordExecutable sub-classes.

### PR DESCRIPTION
**Pull request by EdColeman**
The keyword() method inherited from KeywordExecutable returns a string that was also hard coded into the KeywordStartIT test - use a constant to establish a convention and have the declaration in one place.

### CODE REVIEW

**Review by milleruntime — APPROVED** [review]

File: core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java [hunk 1]
```diff
@@ -83,7 +84,7 @@ public static void main(String[] args) throws Exception {
 
   @Override
   public String keyword() {
-    return "create-empty";
+    return EXE_NAME;
```
Comment (ctubbsii) [inline core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java:87]: I'm not sure I see the benefit of this... it seems to just relocate the constant from an inline literal to a new line that's really only ever needed once (see my subsequent comment about why it's not needed for the test).

File: test/src/main/java/org/apache/accumulo/test/start/KeywordStartIT.java [hunk 2]
```diff
@@ -109,34 +109,35 @@ public void testCheckDuplicates() {
...
-    expectSet.put("wal-info", LogReader.class);
-    expectSet.put("shell", Shell.class);
-    expectSet.put("tserver", TServerExecutable.class);
-    expectSet.put("version", Version.class);
-    expectSet.put("zookeeper", ZooKeeperMain.class);
-    expectSet.put("create-empty", CreateEmpty.class);
-    expectSet.put("split-large", SplitLarge.class);
-    expectSet.put("zoo-zap", ZooZap.class);
+    expectSet.put(Admin.EXE_NAME, Admin.class);
+    expectSet.put(CheckCompactionConfig.EXE_NAME, CheckCompactionConfig.class);
```
Comment (ctubbsii) [inline test/src/main/java/org/apache/accumulo/test/start/KeywordStartIT.java:113]: The whole point of the checks here is to ensure that the string isn't changed. By doing this level of indirection, if the value of the constant `EXE_NAME` is changed, then this test will pass, even if it shouldn't have been changed. This test no longer guards against such changes.
This is now effectively just checking that `SomeClass.EXE_NAME == SomeClass.keyword()` for every `SomeClass`. But, that's basically tautological the way all the classes have changed.

**Review by ctubbsii — CHANGES_REQUESTED** [review]
-1

I don't think this change should be made. It removes a guard against accidental renaming changes or incorrect naming, and effectively tests a tautology.

File: test/src/main/java/org/apache/accumulo/test/start/KeywordStartIT.java (hunk 2 above)
Comment (EdColeman) [reply test/src/main/java/org/apache/accumulo/test/start/KeywordStartIT.java:113]: I'll close this and submit a PR that updates the test documentation to include this. I was unclear on the reason for the test being constructed as it is. When adding / changing a command it is not obvious that the KeywordIT test must also be changed - well, at least until if fails, which is its purpose, but frustrating from a development perspective.

**Comment by EdColeman** [comment]
These changes would invalid the purpose of the test - closing without merging

**Pull request closed without merging**